apps/web/tailwind.config.js

```diff
@@ -174,6 +174,36 @@ export default {
               borderColor: theme("colors.gray.900"),
               borderWidth: "2px",
             },
+            ul: {
```

Comment:
This would be much cleaner with CSS nesting, but we must for now define tailwind-typography prose styles in JS. Waiting for CSS config support for this plugin